src/rust/engine/src/intrinsics.rs

```diff
@@ -517,7 +517,7 @@ fn interactive_process(
       (py_interactive_process.extract().unwrap(), py_process, process_config)
     });
     match process_config.execution_strategy {
-      ProcessExecutionStrategy::Docker(_) | ProcessExecutionStrategy::RemoteExecution => panic!("InteractiveProcess should not set docker_image or remote_execution"),
+      ProcessExecutionStrategy::Docker(_) | ProcessExecutionStrategy::RemoteExecution(_) => panic!("InteractiveProcess should not set docker_image or remote_execution"),
       _ => ()
```

Comment:
This would be better as an error.

Reply:
Agreed. I was being lazy not figuring out how to create the `Failure`. Fixed

Reply:
It doesn't look like these changes have been pushed yet?